app/src/components/Actions/ActionsRow.tsx

```diff
@@ -268,7 +268,7 @@ const SaveFilters = () => {
         setLoading(false);
         socket.send(packageMessage("save_filters", {}));
       }}
-      title={"Save current field filters as view stages"}
+      title={"Mark current field filters as view stages"}
```

Comment:
I'm onboard with the bookmark icon for this action, but I find "mark" in the tooltip a bit confusing. I'd recommend leaving `save` or perhaps `Convert current field filters to view stages` if you want something different